README.md

```diff
@@ -105,11 +106,14 @@ http://owasp.herokuapp.com/
 
 Before you start contributing, please read our brief [style guide](https://github.com/OWASP/owasp-mstg/blob/master/style_guide.md "Style Guide") which contains a few basic writing rules.
 
+Please note that the MSTG focuses primarely on the native apps. These are apps build with Java or Kottlin using the Android SDK for Android or built with Swift or Objective-C using the Apple SDKs for iOS. Nativescript/React-native/Xamarin/Cordova/... apps are not within the focus of the MSTG. However, some key-controls, such as pinning, have been explained already for some of these platforms.
+
 If there's something you really want to see in the guide, or you want to suggest an improvement, create an issue [issue](https://github.com/OWASP/owasp-mstg/issues "Issue") or ping us on [Slack](https://owasp.slack.com/messages/project-mobile_omtg/details/ "Come to our Slack!").
 If the issue is accepted, we will schedule it into one of our [milestones](https://github.com/OWASP/owasp-mstg/milestones "Milestones").
 
 ### Translating
-We are currently in progress of making a new release every 6 months. Releases that have been tagged can then be translated into preferred languages.
+We are currently in progress of making a new minor every 6 months. Next, we will often create patch updates in order to provide intermediary updates in PDF and DocX format. Releases that have been tagged can then be translated into preferred languages.
```

Comment:
We are currently in progress of making a new minor every 6 months -> Our current goal is to publish one minor release every 6 months.

(Or, less freely adapted: We are currently in progress of publishing a new minor release every 6 months)

Reply:
Our current goal is to publish one minor release every 6 months. it is